server/apps/immich/src/main.ts

```diff
@@ -27,6 +27,8 @@ async function bootstrap() {
     app.enableCors();
   }
 
+  const serverPort: number = isNaN(Number(process.env.SERVER_PORT)) ? 3001 : Number(process.env.SERVER_PORT);
```

Comment:
Can you also change this to use the simplified syntax that only uses the env variable once?

Reply:
I have pushed changes for that, I also fixed it in microservices.